ampd/src/event_processor.rs

```diff
@@ -60,7 +59,20 @@ where
             .change_context(Error::EventStream)?;
 
         if let StreamStatus::Active(event) = &stream_status {
-            handler.handle(event).await.change_context(Error::Handler)?;
+            let _ = future::with_retry(
+                || handler.handle(event),
+                // TODO: make timeout and max_attempts configurable
+                RetryPolicy::RepeatConstant {
+                    sleep: Duration::from_secs(1),
+                    max_attempts: 3,
+                },
+            )
+            .await
+            .or_else(|err| {
+                warn!("handler failed to process event {}: {}", event, err);
+
+                Ok::<(), ()>(())
+            });
```

Comment:
It occurred to me that there are probably some errors that we don't want to just ignore, like if signing fails. I'm happy with this for now, but can we add a TODO comment saying we need to find a way to ignore some errors but unwind on others?

Reply:
What is that signing failure?

Reply:
Just any failure within the multisig handler

Reply:
since this is a new design decision and not a shortcoming of the code that might not even be handled any time soon (or ever), I'd prefer not to put a todo in the code. If you want to point attention to it, I'd rather have a comment on line 65 along the line of `// if handlers run into errors we log them and then move on to the next event`